src/room.ts

```diff
@@ -184,6 +192,25 @@ export class Room {
     // Get the local participant
     const p = event.participants["local"];
 
+    // Retrieve our room properties and check if screen sharing is enabled
+    const roomProperties = this.callObject
+      .room({ includeRoomConfigDefaults: true })
+      .then((roomInfo) => {
+        const info = roomInfo as DailyRoomInfo;
```

Comment:
does this need to be reassigned? (I'm guessing it's a typescript thing..)

Reply:
Afraid so; I first tried to cast this to the right type in the callback parameter itself but that didn't work (will look some more tomorrow just in case I missed another way)